### ISSUE TITLE
Filters performances on retina screens

### ISSUE THREAD

**Issue by tristanlemoigne** [pixijs/pixi.js#5726]
Hello, I'm new to PIXI and I would like to add the `GodrayFilter` in my project present in this demo : https://pixijs.io/pixi-filters/tools/demo/

It's working fine for a non-retina device with a pixel ratio of 1 but for a retina device I got my fps dropping to 30 every two frames.I am looking for a way to avoid this behaviour, it's working well on the demo, I checked the source code but didn't find any clue except that the demo can be running at pixel ratio 1. 

Changing the `.resolution` to 1 or 0 on the filter doesn't change anything. 
Changing the application resolution to 1 resolve the problem but affect other sprites quality, I would like to keep it to `window.devicePixelRatio`.
Thanks for your help !

### Expected Behavior
Filter running at 60fps every frames
<img width="181" alt="Capture d’écran 2019-05-26 à 02 08 50" src="https://user-images.githubusercontent.com/28160373/58375771-3f2a5080-7f5b-11e9-946c-815da0e26ad8.png">

### Current Behavior
Filter running at 30fps every two frames
<img width="181" alt="Capture d’écran 2019-05-26 à 02 09 52" src="https://user-images.githubusercontent.com/28160373/58375782-654ff080-7f5b-11e9-884e-ea10df43561e.png">

### Possible Solution
I don't know very much about shaders yet but it can be a problem of my macbook GPU who can't run well this shader on large screen but many users can have this problem too...

### Steps to Reproduce
1. Open http://jellyfish-journey.surge.sh/ on MacBook Chrome
2. Open FPS meter

### Environment
- **`pixi.js` version**: 5.0.3
- **Browser & Version**: Chrome 74
- **OS & Version**: macOS 10.14.4
- **Graphism**: Intel Iris Plus Graphics 640 1536 Mo
- **Running Example**: http://jellyfish-journey.surge.sh/

**Comment by bigtimebuddy** [comment]
Did you try changing PIXI.settings.FILTER_RESOLUTION yet?

**Comment by ivanpopelyshev** [comment]
@bigtimebuddy i believe it affects only constructor

**Comment by tristanlemoigne** [comment]
@bigtimebuddy Yes, thanks I tried it too, this affects well the resolution like `godrayFilter.resolution` but it has no effect when I have a resolution of 2 in my `PIXI.Application` constructor.
Maybe it's a problem of the resolution texture passed to the filter, I tried to modify the `BaseTexture` resolution manually but it didn't work too

**Comment by djmisterjon** [comment]
hum i get strange result on my side, 41% spike on my gpu. it a lot for only one filter! .
![image](https://user-images.githubusercontent.com/24865815/58416074-6f065080-804e-11e9-80f7-d4e012b92f0c.png)

**Comment by tristanlemoigne** [comment]
@djmisterjon yes I have these spikes too and I don't understand where is the problem...
And for medusa, they should not be hidden... I tried on vivaldi browser and I can see them

**Comment by bigtimebuddy** [comment]
I can’t attest to the performance of the godray filter. The community filters are sometimes a mixed bag. It may involve doing some optimization.

**Comment by tristanlemoigne** [comment]
I see, but i don't have the problem on the filters demo, is it displayed only on resolution 1 ? And if yes how the sprites look so good on retina ? mine are blurred 😕

**Comment by alaingalvan** [comment]
Hi, @tristanlemoigne DM'd me (the original author of the shader) to see if there was a way to improve its performance, so I did the following in this pull request [here](https://github.com/pixijs/pixi-filters/pull/213).

1. Using 3D Gradient noise by Inigo Quilez instead of Simplex Noise by Ashima Arts and Stefan Gustavson.

2. Using 4 turbulence layers instead of 6.

3. Since noise functions are much smaller, opting to put everything in `godray.frag` removing `perlin.frag`.

4. There was a typo in the name `gorday.frag`, most likely it was intended to be `godray.frag`, renamed the file to that.

I also updated the original Codepen example to reflect these changes: https://codepen.io/alaingalvan/pen/ICasz
